Create white-spaces.md (#799)
Codecademy/docs: content/general/concepts/white-spaces/whitespace.md

> ---

Reviewer: The Docs folder (currently `white-spaces`) and file name (`whitespace.md`) under must be the same. Let's rename the folder to `whitespace`.

Title: 'Whitespace'
Description: 'Whitespace is spaces and blank lines in programing.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
  - 'Web Development'
Tags:

>   - 'Formatting'

Reviewer: ```suggestion
  - 'Formatting'
  - 'Typography'
```

CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> **Whitespace** is spaces or blank lines in programming. Whitespace is used to format code to improve its readability. 

Reviewer: ```suggestion
In programming, **whitespace** uses horizontal and vertical space characters or blank liens to format code and improve readability.
```

## Example

> The below code is hard to read since it has no gaps and it also does not have indention with spaces so people can not easily see what is nested within what.

Reviewer: ```suggestion
The code example below is hard to read since there is little to no whitespace. It also does not have indention with spaces so people can not easily see what is nested within what.
```

```js
function createRangeIterator(min = 0, max = Infinity, step = 1) {
let nextNum = min;
let numCount = 0;
const rangeIterator = {
next: function () {
let result;
if (nextNum < max) {
result = { value: nextNum, done: false };
nextNum += step;
numCount++;
return result;
}
return { value: numCount, done: true };
},
};
return rangeIterator;
}
```

> When whitespace is used, it is easier to read.

Reviewer: ```suggestion
When whitespace is used, the code is much more readable:
```
